server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -585,6 +585,12 @@ public class ShuffleServerConf extends RssBaseConf {
           .withDescription(
               "keep alive time of thread pool that used for calc summary metric, in SECONDS.");
 
+  public static final ConfigOption<Boolean> APP_LEVEL_SHUFFLE_BLOCK_SIZE_METRIC_ENABLED =
+      ConfigOptions.key("rss.server.block.size.metric.enabled")
```

Comment:
How about `rss.server.blockSizeStatisticsMetrics.enabled`

Reply:
`rss.server.blockSizeStatisticsMetricsEnabled`

Reply:
the name should reflect that this is per-app, not per server